include/wpe/gamepad.h

```diff
@@ -256,6 +256,19 @@ void wpe_gamepad_provider_start(struct wpe_gamepad_provider*);
 WPE_EXPORT
 void wpe_gamepad_provider_stop(struct wpe_gamepad_provider*);
 
+/**
+ * wpe_gamepad_provider_get_implementator_backend:
+ * @provider: opaque libwpe's representation of gamepad provider.
+ *
+ * Called by application (gamepad implementator) to access it's internal object.
+
+ * Returns: the pointer to the implementator's object.
+ *
+ * Since: 1.14
+ */
+WPE_EXPORT
+void* wpe_gamepad_provider_get_implementator_backend(struct wpe_gamepad_provider*);
```

Comment:
This would be “implementer”, but I think the additional word there in the function name is a mouthful. How about calling this just `wpe_gamepad_provider_get_backend()`?

Reply:
done!